mlir/test/mlir-tblgen/attr-or-type-format.mlir

```diff
@@ -129,7 +129,7 @@ func.func private @test_verifier_fails() -> () attributes {
 // -----
 
 func.func private @test_attr_with_type_failed_to_parse_type() -> () attributes {
-  // expected-error@+2 {{invalid kind of type specified}}
+  // expected-error@+2 {{invalid kind of type specified: expected builtin.integer, but found 'vector<4xi32>}}
```

Comment:
It's weird to me that some of the types printed have a leading single quote, some have wrapping single quotes. I suspect that is an issue with the consistency of their printers and not this PR, but I also can't find where in the codebase these single quotes are even emitted. Is it possible you just left off the trailing single quote in your expectation?

Reply:
I meant only the latter. The assertion is `'vector<4xi32>`, not `'vector<4xi32>'`, and in your PR description only a single leading quote is included.

Reply:
ohhh.. that's a typo. You don't have to write the full error message in `expected-error`.